cwms/timeseries/timeseries.py

```diff
@@ -198,7 +190,6 @@ def retrieve_ts_json(
             constants.DATUM: datum,
             constants.BEGIN: begin,
```

Comment:
begin and end need to be isoformatted, otherwise you lose the time zone

Reply:
done